### ISSUE TITLE
trex 0.26.1/0.26.4, driver 512.95 - LHR locked, 512.xx<drivers<512.95 - unstable

### ISSUE THREAD

**Issue by michaelvsinko** [trexminer/T-Rex#1334]
T-Rex: 0.26.4
NVIDIA Driver: 512.95
GPU: rig with GeForce 3060 ti LHR (v2/v3) GPUs
System: Windows 11
ETH Hashrate: 31-32 MH/s
CClock: -200
MClock: 1300
PL: 60

Downgrade to 512.77 helps, Hastrate: 62-64 MH/s

**UP:**
several days of experimenting shown that trex 0.26.4 is unstable. it throws `cuda exception: CUDA_ERROR_ILLEGAL_ADDRESS` approximately 3+ times per hour per gpu. It doesn't depend on overclocking cuz same happening with standard settings and with downclocking
**UP2:**
nbminer 42.2 is stable but has lower hashrate with the same config: 62-64 --> 56-60

**Comment by ifilgud** [comment]
Yeah, 26 Hashrate for 3060. It was 49 on previous driver. It seems they have ""fixed"" the LHR unlock...

**Comment by DIlbert25** [comment]
"Downgrade to 512.77 helps, Hastrate: 62-64 MH/s"

> T-Rex: 0.26.4 NVIDIA Driver: 512.95 GPU: rig with GeForce 3060 ti LHR (v2/v3) GPUs System: Windows 11 ETH Hashrate: 31-32 MH/s CClock: -200 MClock: 1300 PL: 60
> 
> Downgrade to 512.77 helps, Hastrate: 62-64 MH/s

Worked perfectly. Thank you very much.

**Comment by akrueger** [comment]
Same issue for Ergo autolykos

**Comment by sha3rawi33** [comment]
same issue

**Comment by kaizengamer** [comment]
NVIDIA Driver: 512.95
EVGA 3080 FTW3 Ultra
45~50MH/s

Downgraded to 512.15, able to get 101MH/s with some tuning.

**Comment by Raptor2213** [comment]
Thanks for this.  I was getting terrible hashrates on 512.95 as well.  Downgraded to 5.12.77 and it almost doubled my hashrate.  Dev needs to update the changelog to notate this.
